source/dart/lib/astronomy.dart

```diff
@@ -1,7 +1,24 @@
 library astronomy;
 
+import 'dart:core';
 import 'dart:math';
 
+import 'package:meta/meta.dart';
+
+/// The number of minutes in a day.
+const minutesPerDay = 60.0 * 24.0;
+
+/// The number of seconds in a day.
+const secondsPerDay = 60.0 * minutesPerDay;
+
+/// The number of milliseconds in a day.
+const millisecondsPerDay = 1000.0 * secondsPerDay;
+
+const daysPerTropicalYear = 365.24217;
+const dayPerMillennium = 365250.0;
```

Comment:
As `Prefer using lowerCamelCase for constant names. (Documentation)` https://dart-lang.github.io/linter/lints/constant_identifier_names.html